corehq/apps/hqadmin/tasks.py

```diff
@@ -180,3 +181,25 @@ def track_es_doc_counts():
                     }
                     metrics_gauge('elasticsearch.shards.docs.count', i['docs']['count'], tags)
                     metrics_gauge('elasticsearch.shards.docs.deleted', i['docs']['deleted'], tags)
+
+
+@periodic_task(queue='background_queue', run_every=crontab(minute="0", hour="0"))
+def track_pg_limits():
+    for db in settings.DATABASES:
+        with connections[db].cursor() as cursor:
+            query = """
+            select tab.relname, seq.relname, attlen
+              from pg_class seq
+              join pg_depend dep on seq.oid=dep.objid
+              join pg_class as tab on (dep.refobjid = tab.oid)
```

Comment:
nit: any reason there are parentheses and spaces around the `=` here but not on the other lines?

Reply:
nope, i can remove those. As I was building this query I added them and then never made everything consistent at the end.